consensus/istanbul/backend/handler.go

```diff
@@ -201,8 +201,6 @@ func (sb *Backend) SetP2PServer(p2pserver consensus.P2PServer) {
 func (sb *Backend) NewWork() error {
 	sb.logger.Debug("NewWork called, acquiring core lock", "func", "NewWork")
 
-	sb.coreMu.RLock()
-	defer sb.coreMu.RUnlock()
```

Comment:
This removal looks safe to me, since it just locks over access of `coreStarted` and a logging statement.